Kernel/FileSystem/Inode.cpp

```diff
@@ -232,6 +232,12 @@ void Inode::did_remove_child(InodeIdentifier, StringView name)
 
 void Inode::did_modify_contents()
 {
+    // FIXME: What happens if this fails
```

Comment:
There is a missing `?` here I'd say, its presence would help a bit with reading.